docs/provider-1password-automation.md

```diff
@@ -0,0 +1,172 @@
+## 1Password Secrets Automation
+
+External Secrets Operator integrates with [1Password Secrets Automation](https://1password.com/products/secrets/) for secret management.
+
+### Important note about this documentation
+_**The 1Password API calls the entries in vaults 'Items'. These docs use the same term.**_
+
+### Behavior
+* How an Item is equated to an ExternalSecret:
+    * `remoteRef.key` is equated to an Item's Title
+    * `remoteRef.property` is equated to:
+        * An Item's field's Label (Password type)
+        * An Item's file's Name (Document type)
+        * If empty, defaults to the first file name, or the field labeled `password`
+    * One Item in a vault can equate to one Kubernetes Secret to keep things easy to comprehend.
```

Comment:
We should add here that `remoteRef.version` is currently not supported, as well as return a `Not Supported` error in the `GetSecret()` and `GetSecretMap()` methods

Reply:
Will do..

Reply:
I have now done both these things (readme and not supported check/error in code.)